evgMrmApp/Db/Makefile

```diff
@@ -25,7 +25,7 @@ DB += evgDbus.db
 DB += evgAcTrig.db
 DB += evgInput.db
 DB += evgOutput.db
-DB += evgSoftSeq.db
+DB += evgSoftSeq.template
```

Comment:
I would use ```evgSoftSeq.db``` in the installation. For what do you need that?

Reply:
evgSoftSeq.db has implemented the default substitutions. When I am trying to build the database for our APS application, I found that I did need the PP macro.